uniffi/src/ffi/handle_map.rs

```diff
@@ -0,0 +1,243 @@
+use ffi_support::ExternError;
+use ffi_support::Handle;
+use ffi_support::HandleError;
+use ffi_support::HandleMap;
+use ffi_support::IntoFfi;
+use std::sync::RwLock;
```

Comment:
This file is copied from the `ffi_support::handle_map::ConcurrentHandleMap`, and edited.

TODO:

 * test test test. I haven't yet copied over the tests, and I don't know what will still make sense.
 * re-write/edit the documentation,